src/cmd-handlers.ts

```diff
@@ -0,0 +1,108 @@
+function cmd_handlers(initialized_req_handler: CallableFunction, body_specs: Record<string, string> = {}, include?: string[], query_params?: Record<string, any>) {
+  async function handler(this:any, msg: any) {
+    let request_body: any = {}
+    let source: any
+    let args: Record<string, any>
+    let path_args: Record<string, any>
+    let method
+
+    switch (msg.cmd) {
```

Comment:
as with github-provider, the code structure we want to use here is higher order functions that construct the load or save action (there will be other actions later, so we need a codebase that is easy to extend)